tools/cleed_io/writer.py

```
@@ -0,0 +1,55 @@
+            f.write(format_atom("pb", atom, phase_map))
+            
+        # Add standard params if missing (users should edit these)
+        f.write("\n# Standard Parameters (Adjust as needed)\n")
+        f.write("vr: -10.0\n")
+        f.write("vi: 4.0\n")
+
+def write_inp(filename, overlayer_atoms: Atoms, bulk_atoms: Atoms, phase_map=None):
+    """
+    Writes the CLEED .inp file.
+    """
+    cell = bulk_atoms.get_cell() # Overlayer uses bulk 2D periodicity usually
```

Ruff (0.14.10)

Local variable `cell` is assigned to but never used

Remove assignment to unused variable `cell`

(F841)